composer/algorithms/ema/ema.py

```diff
@@ -111,154 +116,172 @@ class EMA(Algorithm):
             )
     """
 
-    def __init__(self, half_life: str, update_interval: Optional[str] = None, train_with_ema_weights: bool = False):
-        self.half_life = half_life
-        self.update_interval = update_interval
-        self.train_with_ema_weights = train_with_ema_weights
-
+    def __init__(self,
+                 half_life: Optional[str] = None,
+                 smoothing: Optional[float] = None,
```

Comment:
Do we want one of these to have a valid default value?

Reply:
Ah yeah good question. If I were to set one, I'd lean towards setting a default of `half_life="1000ba"`. I didn't set it because if someone wants to set the value of `smoothing` and forgets about the default `half_life`, then they'd get an error message. I'm not sure this is the right decision though, as now they'll get an error if neither one is specified. What do you think?

Reply:
Ok, added a valid default to `half_life`